scripts/fdbshow

```diff
@@ -93,7 +93,24 @@ class FdbShow(object):
         self.bridge_mac_list.sort(key = lambda x: x[0])
         return
     
-    
+    def fetch_state_fdb_data(self):
```

Comment:
>fetch_state_fdb_data [](start = 8, length = 20)

@kcudnik Please help check.
The StateDB FDB is learned from syncd notification. So is it the same as AsicDB, for example, how about static fdb entries programmed?

Reply:
yes, asic db learns those from fdb notification, static entries are created by user explicitly, so those comes from OA

Reply:
Yes, It does not process static fdb tables.  It is a problem.

Reply:
If you cannot find static FDB entries in StateDB, it will be inside AppDB. FYI if you want to further explore.

---
In reply to: [450189315](https://github.com/Azure/sonic-utilities/pull/968#discussion_r450189315) [](ancestors = 450189315)

Reply:
So, thats a good reason to have an option to display from state_db based on need. By default, and for small number of entries, it can use the existing option. But for large number, user can specify to display from state_db

Reply:
It is a good  idea. I will add the option for the command.

Reply:
I have add option "-f" to for displaying large number of FDB entries fastly from state_db and appl_db. 
The static FDB entries are also displayed from appl_db in fast option.

root@sonic:/home/admin# fdbshow -h
usage: fdbshow [-h] [-p PORT] [-v VLAN] [-f]

Display ASIC FDB entries

optional arguments:
  -h, --help            show this help message and exit
  -p PORT, --port PORT  FDB learned on specific port: Ethernet0
  -v VLAN, --vlan VLAN  FDB learned on specific Vlan: 1001
  -f, --fast            Display large number of FDB entries fastly
root@sonic:/home/admin# 
root@sonic:/home/admin# fdbshow -f
  No.    Vlan  MacAddress         Port       Type
-----  ------  -----------------  ---------  -------
    1     100  00:00:00:00:00:01  Ethernet1  dynamic
    2     100  11:00:00:00:00:01  Ethernet1  static
    3     100  11:00:00:00:00:02  Ethernet1  static
    4     100  11:00:00:00:00:03  Ethernet1  static
Total number of entries 4